Assets/_ARPlane/Scripts/UI/RadarUI.cs

```diff
@@ -0,0 +1,42 @@
+﻿using UnityEngine;
+using UniversoAumentado.ARCraft.AR;
+
+public class RadarUI : MonoBehaviour
+{
+    [SerializeField] private ARCraft arCraft;

```

Comment:
Can probably be any GameObject? So it could also for e.g. incoming missiles, other planes, etc?

Reply:
Well could be but there are no incoming missiles at this moment. Also I imagine that that would be a different monobehaviour (it would probably act a bit different, like it would show distance, flashing warning signals and what have you). Rather than to provide provision for every future scenario let's go what with works now and 

![image](https://user-images.githubusercontent.com/985892/84122845-568a8d80-aa39-11ea-9214-e1f1e6b80f00.png)

Reply:
There is no logic that is specific for ARCraft here, you're only using transforms. So why make it a requirement that it's an ARCraft? Showing distance, flashing warnings, etc can also be different components (the advantage of component-based programming). I would definitely not want to copy the screen math to a separate monobehaviour.

But whatever, go ahead and just ship it.

Reply:
Okay you have a point im not actually needing anything specific to arCrafts. Updated PR.